shortfin/CMakeLists.txt

```diff
@@ -118,11 +118,16 @@ option(SHORTFIN_ENABLE_ASAN "Enable ASAN" OFF)
 if(SHORTFIN_ENABLE_ASAN)
   add_compile_options(-fsanitize=address)
   add_link_options(-fsanitize=address)
-
-  # Enable more ASAN checks.
   add_compile_definitions(IREE_SANITIZER_ADDRESS)
 endif()
 
+option(SHORTFIN_ENABLE_TSAN "Enable TSAN" OFF)
```

Comment:
Can you add a note here advertising the TSAN option too? https://github.com/nod-ai/shark-ai/blob/26bb2505e5dfa6bcde6e4d10a41f584ba1704a48/shortfin/README.md?plain=1#L146-L148

Reply:
I haven't tried to run anything else than `ctest` so I have no experience with Python so I wouldn't know what to write here.... can I just defer to you in a follow-up?

Reply:
Ah, fair. That is Python-specific and does also rely on this plumbing: https://github.com/nod-ai/shark-ai/blob/26bb2505e5dfa6bcde6e4d10a41f584ba1704a48/shortfin/setup.py#L228

I was mainly looking for any developer docs we have that mention ASan, since they could also mention TSan and other sanitizers too.

Fine to just add the option to CMake for now.